adcm_aio_client/core/objects/_accessors.py

```diff
@@ -19,12 +19,13 @@
 from adcm_aio_client.core.types import Endpoint, QueryParameters, Requester, RequesterResponse
 
 
-class Accessor[ReturnObject: InteractiveObject, Filter](ABC):
+class Accessor[ReturnObject: InteractiveObject, Filter: dict | None](ABC):
     class_type: type[ReturnObject]
 
-    def __init__(self: Self, path: Endpoint, requester: Requester) -> None:
```

Comment:
I suggest thinking about naming filters argument somehow different to specify that it'll be used for each request in priority to user-defined

Ideas are: default_filter(s), narrowing_filter

---

Type is dict | None, since you set None as default

---

Also note that it should be high level filter OR named somehow different from "filter", because filter mechanism is our "high-level" mechanism to working with client

Reply:
renamed to `accessor_filter`

Reply:
fix typehint pls, |None

Reply:
its declared earlier `type AccessorFilter = QueryParameters | None`